packages/lit/src/components/font-select/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { beforeEach, describe, expect, test } from 'vitest';
+import './index';
+
+const tag = 'font-select';
+
+describe(`<${tag}>`, () => {
+  beforeEach(() => {
+    document.body.innerHTML = `<${tag}></${tag}>`;
+  });
+
+  test('shows a select element', async () => {
+    const element = document.querySelector(tag);
+    const select = element?.shadowRoot?.querySelector('select');
+    expect(select).toBeDefined();
+  });
+
+  test('shows a list of common fonts', async () => {
+    const element = document.querySelector(tag);
+    const options = element?.shadowRoot?.querySelectorAll('option');
+    expect(options?.length).toBeGreaterThan(1);
+  });
+});
```

Comment:
Het zou ook mooi zijn als je `getByRole('option')` enzo kan doen op basis van de accessibility tree. Volgens mij heeft Tim al een library aan de praat die dat kan doen in de shadow roots.